scripts/amd/gemm/tune_gemm.py

```diff
@@ -348,30 +389,33 @@ def main():
         f_kernel[fi].close()
 
 
-def extract_kernel_time(M, N, K, config, df):
-    configStr, _ = gen_kernel_and_configStr_from_config(M, N, K, config, None, None, None)
+def extract_kernel_time(M, N, K, config, df, bias_size):
+    configStr, _ = gen_kernel_and_configStr_from_config(M, N, K, config, None, None, None, bias_size)
     df = df[df['KernelName'].str.contains(configStr)]
     meanTime = df['DurationNs'].tail(100).mean()
     return config, meanTime
 
 
-def profile_batch_kernels(M, N, K, gpuid, gpus, jobs, verbose):
+def profile_batch_kernels(M, N, K, gpuid, gpus, jobs, verbose, rotating_buffer_size):
     ngpus = len(gpus)
     gpuIdx = gpus.index(gpuid)
     if gpuIdx + 1 > jobs:
         return
     os.environ['ROCR_VISIBLE_DEVICES'] = str(gpuid)
     jobId = gpuIdx
     while jobId < jobs:
+        kernel_name = generated_kernel_name(M, N, K, jobId)
         if verbose:
             print(f"profiling {generated_kernel_name(M, N, K, jobId)} on GPU {gpuid}")
-        run_bash_command_wrapper(f"rocprof --stats -o results-{jobId}.csv python {generated_kernel_name(M, N, K, jobId)}", capture=(verbose < 2))
+        run_bash_command_wrapper(f"rocprof --stats -o results-{kernel_name}.csv python {kernel_name} -rotating_tensor {rotating_buffer_size}", capture=(verbose < 2))
```

Comment:
Now the `{kernel_name}` includes the absolute path of the file, e.g.
```bash
rocprof --stats -o results-/var/lib/jenkins/triton/xxx/yyy/zzz/generated_kernel.py.csv ... 
```
And rocprof does not work. I'd suggest we keep the old `{jobId}`.